src/main/java/uk/gov/ons/census/casesvc/utility/Sha256Helper.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ons.census.casesvc.utility;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class Sha256Helper {
+  private static MessageDigest digest;
+
+  static {
+    try {
+      digest = MessageDigest.getInstance("SHA-256");
+    } catch (NoSuchAlgorithmException e) {
+      throw new RuntimeException();
+    }
+  }
+
+  public static String hash(String stringToHash) {
+    byte[] encodedhash = digest.digest(stringToHash.getBytes(StandardCharsets.UTF_8));
+
+    return (bytesToHex(encodedhash));
+  }
+
+  private static String bytesToHex(byte[] hash) {
+    StringBuffer hexString = new StringBuffer();
```

Comment:
Can we use some like this instead:
return javax.xml.bind.DatatypeConverter.printHexBinary(hash).toLowerCase();

I think it does the job and is less hand rolled (well mkyong rolled anyway)